src/index.ts

```diff
@@ -21,6 +21,9 @@ function pluginEntry(pluginOptions?: Partial<BuildCesiumOptions>): Plugin[] {
     importCesium(base => `${base}${options.from}Unminified/Cesium.js`, 'serve'),
     importCesium(base => `${base}${options.to}/Cesium.js`, 'build'),
 
+    importCss(options, base => `${base}${options.from}Unminified/Widgets/widgets.css`, 'serve'),
```

Comment:
Is there any particular reason to use `Unminified` `css`? I use `Unminified` `Cesium.js` because it could throw readable error message. But it seems unnecessary to use unminified style.

Reply:
You're right, no need to use unminified css!